src/tsConfigNormalizer.ts

```diff
@@ -29,16 +30,17 @@ export function normalizeTsConfig(input: string, altBase: string): string {
   return output;
 }
 
-function isJSFile(file: string) {
-  return file.endsWith('.js');
-}
-
 export function load(file: string): RawConfigFile {
-  if (isJSFile(file)) {
-    return require(file);
-  }
+  // Check if its a .js file, use require
+  if (file.endsWith('.js')) return require(file);
```

Comment:
Thanks very much for your PR, and this will be a really useful feature.  However, we don't have to inline method `isJSFile`.

Here we have a new comment, but the method signature was already self-explanatory.  I would choose to let the code itself explain its intention since it eventually makes more concise and maintainable codebase.

I would appreciate it if these two lines could be reverted.  Thank you!

Reply:
Yea no problem! The only reason I inlined the code was because the function was one line and was only used in that function. Would you like me to extract the yaml check as well? Its roughly the same length of code?

Reply:
Thanks for your understanding 🙇  Actually, this method was not written by me, but I think this method expresses domain logic eloquently though not reused elsewhere.

And thank you for your suggestion, yes I agree with that.  YAML check can be extracted as well :-)

Reply:
Updated!

Reply:
👍👍👍👍